policyengine_us/variables/gov/states/vt/tax/income/adjusted_gross_income/additions/vt_agi_additions.py

```diff
@@ -0,0 +1,26 @@
+from policyengine_us.model_api import *
+
+
+class vt_agi_additions(Variable):
+    value_type = float
+    entity = TaxUnit
+    label = "VT AGI additions"
+    unit = USD
+    documentation = "Additions to VT AGI over federal AGI."
+    definition_period = YEAR
+    defined_for = StateCode.VT
+    reference = [
```

Comment:
for references in variables we only need the links - [example](https://github.com/PolicyEngine/policyengine-us/blob/master/policyengine_us/variables/gov/states/ne/tax/income/ne_eitc.py)